pfb-analysis/scripts/entrypoint.sh

```diff
@@ -52,6 +52,18 @@ then
     popd
 fi
 
+# If given a URL for the OSM file, dowload and unzip it. Overrides PFB_OSM_FILE.
+if [ "${PFB_SHPFILE_URL}" ]
```

Comment:
Comment says this is to download OSM file, but parameter checked is for shapefile? Below it references an OSM URL parameter, but downloads it as a zip. Only the shapefile is zipped. There are two things to download, an OSM file and a zipped shapefile. The shapefile is the one needing extraction.